docs/code/MlNetCookBook.md

```diff
@@ -797,10 +776,10 @@ Sentiment   SentimentText
 ```csharp
 // Create a new environment for ML.NET operations. It can be used for exception tracking and logging, 
 // as well as the source of randomness.
```

Comment:
Would be a good idea to replace with the above comments:

// Create a new context for ML.NET operations. It can be used for exception tracking and logging, 
// as a catalog of available operations and as the source of randomness.